src/ert/config/workflow_job.py

```diff
@@ -156,10 +163,14 @@ def __post_init__(self) -> None:
                 f"type, expected ErtScript, got {self.ert_script}"
             )
 
-        self.source_package = self.ert_script.__module__.partition(".")[2]
         self.description = (
             self.description or self.ert_script.__doc__  # type: ignore
         )
+        return self
+
+    @computed_field
+    def source_package(self) -> str:
```

Comment:
Do we need this as a field? Can this become just a property instead?

Reply:
Sorry, think I forced pushed, I converted it to property